pandas/core/generic.py

```diff
@@ -3141,6 +3146,9 @@ def to_csv(
 
         from pandas.io.formats.csvs import CSVFormatter
 
+        kwargs.setdefault("lineterminator", line_terminator)
+        line_terminator = line_terminator or kwargs["lineterminator"]
```

Comment:
need to watch out for corner case where user passes both

Reply:
Ok! Do we throw an error or would we use one (maybe `line_terminator`) and a throw warning?